configure.ac

```diff
@@ -172,6 +172,16 @@ AS_IF([test "${with_platform}" = "linux-generic"],
 AC_DEFINE_UNQUOTED([IMPLEMENTATION_NAME], ["$IMPLEMENTATION_NAME"],
 		   [Define to the name of the implementation])
 
+##########################################################################
+# Build examples/tests dynamically
+##########################################################################
+AC_ARG_ENABLE([static-link],
+	      [AS_HELP_STRING([--disable-static-link],
+			      [disable static linking of examples and tests])],
+	      [static_link=$enableval],
+	      [static_link=yes])
+AM_CONDITIONAL([STATIC_LINK], [test "x$static_link" != "xno"])
+
```

Comment:
At his time if --disable-static-link is specified then linking is static. If it's not specified that it's dynamic. I'm looking on odp_generator.

Reply:
Well, no. Please recheck. I will update description to mention that is means static linking of libodp-linux into examples.